docs/integrations/ember-and-json-api.md

```diff
@@ -117,7 +119,7 @@ will generate the path `/posts/{postId}?include='comments'`
 
 So then in your controller, you'll want to be sure to have something like:
 ```ruby
-render json: @post, include: params[:include]
+render jsonapi: @post, include: params[:include]
```

Comment:
@Patsy-issa My only other comment, and maybe we should discuss this in a PR, is that it's generally not a good idea to blindly pass in query params like `params[:include]`.  You should instead use a whitelist of valid params, like `params.require(:include).permit(:author, :comments)`